app/controllers/project_instances_controller.rb

```diff
@@ -59,9 +59,12 @@ def update
   end
 
   def new
-    project_id = Project.where("url like ?", "%#{params[:nagios_url]}%").first.try(:id) if params[:nagios_url]
+    nagios_url = params[:nagios_url].presence || ""
```

Comment:
Ops, I forgot to use `.present?` on nagios_url!
I used the `|| ""` also because I can prepend the 'https://' string by default if there is no `:nagios_url` param, but maybe I can skip that